### PR TITLE
doc: addendum to 12.1.0 release announcements

### PR DESCRIPTION
**Pull request by theanalyst**
Added mon, rbd and an update to the RGW section

Signed-off-by: Abhishek Lekshmanan <alekshmanan@suse.com>

### CODE REVIEW

**Comment by theanalyst** [comment]
ideally should've  been in https://github.com/ceph/ceph/pull/15626 but that was merged

**Comment by mattbenjamin** [comment]
@theanalyst thanks!

File: doc/release-notes.rst [hunk 1]
```diff
@@ -107,6 +120,10 @@ Major Changes from Kraken
   * RGW has consolidated the several metadata index pools via the use of rados
     namespaces
   * RGW now supports dynamic bucket index sharding
+  * AWS like bucket policy api support, for now, policy is a means to
```
Comment (theanalyst) [inline doc/release-notes.rst]: ack, updating

File: doc/release-notes.rst [hunk 2]
```diff
@@ -91,10 +91,25 @@ Major Changes from Kraken
   * The "ceph mds tell ..." command has been removed.  It is superceded
     by "ceph tell mds.<id> ..."
 
+- *MON*
+  * Ability to replace osds via `osd destroy` & `osd new` commands
```
Comment (Liuchang0812) [inline doc/release-notes.rst]: Insert a blank line between lines 94 and 95

**Comment by xiexingguo** [comment]
s/11.1.0/12.1.0

File: doc/release-notes.rst (hunk 2 above)
Comment (theanalyst) [inline doc/release-notes.rst]: thanks!

File: doc/release-notes.rst [hunk 3]
```diff
@@ -91,10 +91,26 @@ Major Changes from Kraken
   * The "ceph mds tell ..." command has been removed.  It is superceded
     by "ceph tell mds.<id> ..."
 
+- *MON*
+
+  * Ability to replace osds via `osd destroy` & `osd new` commands
+  * `osd purge` command added which combines the functionality of
+    `osd destroy`, `osd rm` and `osd crush rm`
+
 - *MGR*
```
Comment (Liuchang0812) [inline doc/release-notes.rst]: ditto @theanalyst

**Comment by liewegas** [comment]
incorported this into the other PR

**Pull request closed without merging**